src/main/java/com/amadeus/resources/FlightAvailabilitySearch.java

```diff
@@ -0,0 +1,64 @@
+package com.amadeus.resources;
+
+import com.amadeus.shopping.FlightAvailabilitiesSearch;
+import com.google.gson.annotations.SerializedName;
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * A FlightAvailabilitySearch object as returned by the FlightAvailabilitiesSearch API.
+ * @see FlightAvailabilitiesSearch#get()
+ */
+@ToString
+public class FlightAvailabilitySearch extends Resource {
+  protected FlightAvailabilitySearch() {}
+
+  private @Getter String type;
+  private @Getter String id;
+  private @Getter String originalDestinationId;
+  private @Getter String source;
+  private @Getter Boolean instantTicketingRequired;
+  private @Getter Boolean paymentCardRequired;
+  private @Getter String duration;
+  private @Getter SearchSegment[] segments;
+
+  @ToString
+  public class SearchSegment {
```

Comment:
I think you have missed some elements from the reply object.
I can't see the closedStatus, co2Emissions, operating, duration and stops

Reply:
Thank you for noticing that! I have added the missed elements by following the model in the [doc](https://developers.amadeus.com/self-service/category/air/api-doc/flight-availabilities-search/api-reference).